linkerd/app/core/src/metrics.rs

```diff
@@ -266,6 +269,8 @@ impl FmtLabels for InboundEndpointLabels {
             write!(f, ",")?;
         }
 
+        write!(f, "target_addr=\"{}\"", self.target_addr)?;
```

Comment:
I would think this needs a trailing comma? Surprised our tests pass...

Reply:
Yeah, commit c75b2d5d835615fd77f8a978c2fd5113af1fc3e4 updates the metrics test code to actually check that metric labels are formatted correctly --- the tests now fail with errors like
```
thread 'tests::telemetry::metrics_endpoint_inbound_request_count' panicked at 'malformed metric (multiple '=' in label)
 in label: "target_addr=\"127.0.0.1:34111\"tls=\"disabled\""
   metric: "request_total{direction="inbound",authority="tele.test.svc.cluster.local",target_addr="127.0.0.1:34111"tls="disabled"} 1"
  retried 5 times (73.354973ms)', linkerd/app/integration/src/metrics.rs:194:51
```

Good catch!